modules/apps/callbacks/callbacks_test.go

```diff
@@ -22,6 +27,25 @@ import (
 
 const maxCallbackGas = uint64(1000000)
 
+func init(){
+	ibctesting.DefaultTestingAppInit = func() (ibctesting.TestingApp, map[string]json.RawMessage) {
+		db := dbm.NewMemDB()
+		encCdc := simapp.MakeTestEncodingConfig()
+		app := simapp.NewSimApp(log.NewNopLogger(), db, nil, true, simtestutil.EmptyAppOptions{})
+		return app, simapp.NewDefaultGenesisState(encCdc.Codec)
+	}
+}
+
+// GetSimApp returns the duplicated SimApp from within the callbacks directory.
+// This must be used instead of chain.GetSimApp() for tests within this directory.
+func GetSimApp(chain *ibctesting.TestChain) *simapp.SimApp {
+	app, ok := chain.App.(*simapp.SimApp)
+	if !ok {
+		panic("chain is not a simapp.SimApp")
+	}
+	return app
+}
```

Comment:
For context we essentially went with the same way the ica-demo repo wired up the testing app.

Links:
- https://github.com/cosmos/interchain-accounts-demo/blob/master/x/inter-tx/keeper/keeper_test.go#L35-L44
- https://github.com/cosmos/interchain-accounts-demo/blob/master/x/inter-tx/keeper/keeper_test.go#L98-L105